test/ctia/test_helpers/core.clj

```diff
@@ -139,3 +146,21 @@
   (id/->id type-kw
            (crud/make-id (name type-kw))
            (get-in @properties [:ctia :http :show])))
+
+(defn fixture-spec-validation [t]
+  (with-redefs [cs/registry-ref (atom (cs/registry))]
+    (cs/check-asserts true)
+    (t)
```

Comment:
does it means we need to enable spec validation on prod deployment to validate references?